course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -105,12 +105,24 @@
                 });
             };
 
+            $scope.getCourseDescription = function(course) {
+                // If a course's title is [NULL], attempt to display the short title.
+                // If the short title is also [NULL], display [School] 'Untitled Course' [Term Display]
+                if(typeof course.title != "undefined" && course.title.length > 0){
+                    return course.title;
+                }
+                else if(typeof course.short_title != "undefined" && course.short_title.length > 0){
+                    return course.short_title;
+                }
+                return 'Untitled Course';
```

Comment:
I thought that, but then I realized that the two columns on either side of title are School and Term. I did make an assumption that that's what they meant. I will double check thought, you might be right.